samples/msal-node-samples/client-credentials/index.js

```diff
@@ -7,10 +7,19 @@ var msal = require('@azure/msal-node');
 
 const config = {
     auth: {
-        clientId: "12d77c73-d09d-406a-ae0d-3d4e576f7d9b",
-        authority: "https://login.microsoftonline.com/90b8faa8-cc95-460e-a618-ee770bee1759",
-        clientSecret: ""
-   } 
+        clientId: "48f55727-da76-4dbb-9908-d9335f239e20",
+        authority: "https://login.microsoftonline.com/eefab79a-7793-4852-be01-105ecb6f6d4e/",
+        clientSecret: "epFuS.3-QDTU3DGI-QQs8UwQx-34jy.u9y",
```

Comment:
This should not be open. Can we share it in keyvault and programmatically read it? cc @derisen

Reply:
Hi @sameerag, don't we have all samples sharing their client information publicly and if we put in in a key vault wouldn't users of the sample be able to access the details all the same?

Reply:
Anyone using samples that require secrets must provide their own. Please remove this, and invalidate the secret from the app registration.

Reply:
I have removed them and will invalidate them.